www/gatsby-config.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+module.exports = {
+  siteMetadata: {
+    title: 'React: A JavaScript library for building user interfaces',
+  },
+  mapping: {
+    'MarkdownRemark.frontmatter.author': 'AuthorYaml',
+  },
+  plugins: [
+    'gatsby-transformer-authors-yaml',
+    'gatsby-plugin-netlify',
+    'gatsby-plugin-glamor',
+    'gatsby-plugin-react-next',
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        path: `${__dirname}/src/pages`,
+        name: 'pages',
+      },
+    },
+    {
+      /* Docs, Tutorial, Community, and Blog section content comes from here. */
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `packages`,
+        path: `${__dirname}/../docs/`,
+      },
+    },
+    {
+      resolve: `gatsby-transformer-remark`,
+      options: {
+        plugins: [
+          'gatsby-remark-responsive-iframe',
+          {
+            resolve: `gatsby-remark-images`,
+            options: {
+              maxWidth: 800,
+            },
+          },
+          'gatsby-remark-autolink-headers',
+          'gatsby-remark-use-jsx',
+          {
+            resolve: `gatsby-remark-prismjs`,
+            options: {
+              classPrefix: 'gatsby-code-',
+            },
+          },
+          'gatsby-remark-copy-linked-files',
+          'gatsby-remark-smartypants',
+        ],
+      },
+    },
+    `gatsby-transformer-sharp`,
+    `gatsby-plugin-sharp`,
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        //trackingId: `ADD YOUR TRACKING ID HERE`,
```

Comment:
This needs copied over from the existing site (I'm assuming you're using Google Analytics — if not remove this)